contrib/compose-rhcs.sh

```diff
@@ -6,7 +6,16 @@ set -e
 # VARIABLES #
 #############
 
-STAGING_DIR=staging/main-ubi9-latest-x86_64/
+if [[ -z "${UBI_VERSION}" ]]; then
+  echo "ERROR: UBI_VERSION must be set! (eg: UBI_VERSION=9)"
+  exit 1
+fi
+if [[ -z "${CEPH_RELEASE}" ]]; then
+  echo "ERROR: CEPH_RELEASE must be set (eg: CEPH_RELEASE=quincy)"
+  exit 1
+fi
```

Comment:
Something like this:
```suggestion
if [[ -z "${VERSION}" ]]; then
  echo "ERROR: VERSION must be 5 or 6"
  exit 1
elif [ "$VERSION" == "5" ]; then
  UBI_VERSION=8
  CEPH_RELEASE=pacific
elif [ "$VERSION" == "6" ]; then
  UBI_VERSION=9
  CEPH_RELEASE=quincy
else
  echo "ERROR: VERSION must be 5 or 6"
  exit 1
fi
```